apps/emqx_plugin_libs/src/emqx_trace/emqx_trace.erl

```diff
@@ -153,7 +156,7 @@ update(Name, Enable) ->
             [] -> mnesia:abort(not_found);
             [#?TRACE{enable = Enable}] -> ok;
             [Rec] ->
-                case erlang:system_time(second) >= Rec#?TRACE.end_at of
+                case os:system_time(second) >= Rec#?TRACE.end_at of
```

Comment:
nit: Could you wrap all `os:system_time/1` calling into a function? And write a clear comment as to why `os:system_time/1` is used instead of erlang:system_time/1 like other modules. Otherwise, I'm afraid we'll forget why this is needed in the future.

Reply:
good point